benchmarks/OpOptimization/BatchNormalization/TVM/CPU/batchNormalization_manual.py

```diff
@@ -0,0 +1,158 @@
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# ===---------------------------------------------------------------------------
+#
+# This file implements the manual optimization for BatchNormalization on CPU.
+# Autoscheduler is TVM's next-generation performance tuning tool,
+# which can automatically generate search spaces for optimizing tensor expressions.
+# TVM is an Apache-2.0 licensed project.
+# See the TVM license at: https://github.com/apache/tvm/blob/main/LICENSE
+#
+# ===---------------------------------------------------------------------------
+import tvm
+from tvm import te
+import numpy as np
+from tvm import topi
+import timeit
+import mxnet as mx
```

Comment:
Please give instructions on compiling mxnet, the package from pypi is not working sometimes.